rust/tool/shared/src/signature/README.md

```diff
@@ -0,0 +1,34 @@
+# Signatures capabilities of Lanzaboote
```

Comment:
This documentation is nice, but it looks like it should be the module documentation for the signature module and not a standalone README. Part of this can also go away, because it duplicates documentation of the `LanzabooteSigner` trait.